homeassistant/components/tplink/sensor.py

```diff
@@ -69,6 +74,81 @@ class TPLinkSensorEntityDescription(
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    TPLinkSensorEntityDescription(
+        # Disable as the value reported by the device changes seconds frequently
+        entity_registry_enabled_default=False,
+        key="on_since",
+        device_class=SensorDeviceClass.TIMESTAMP,
+    ),
+    TPLinkSensorEntityDescription(
+        key="rssi",
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    TPLinkSensorEntityDescription(
+        key="signal_level",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    TPLinkSensorEntityDescription(
+        key="ssid",
+    ),
+    TPLinkSensorEntityDescription(
+        key="current_firmware_version",
+    ),
+    TPLinkSensorEntityDescription(
+        key="available_firmware_version",
+    ),
```

Comment:
Should we also disable these, pending the `update` platform?

Reply:
Done